### DISCUSSION TITLE
The label mcmc results diagram

### DISCUSSION THREAD

**Discussion by d11122** [phoebe-project/phoebe2#468] [Questions]
hello, I have a question in my work.
I want to change the labels in the mcmc result graph to what I want instead of the default. For example change Rquiv@primary to R2. What should I do?

in this file :'Advanced: Parameter Latex Representation' ，I can only change the label of the distribution.

I look forward to your reply！

Thank you!
Daimin

**Comment by gecheline** [comment]
Hi Daimin,

Phoebe will also let you change 'primary' to 1 and 'secondary' to 2 if you follow the Parameter Latex Representation tutorial, however I don't think you can remove the 'equiv' from Requiv. I've had to do something similar and I usually do it by accessing the matplotlib object returned when calling b.plot(). Here's some pseudocode:
`
afig, mplfig = b.plot(solution={your solution}, style='corner', show=True)
mplfig.axes[{index}].set_xlabel('{your label}')
`
You can then save the updated mplfig object with mplfig.savefig

**Reply by d11122** [reply]
Dear Angela Kochoska，thank you for your reply! I will try it with your idea.
